Pickers/AbstractActionSheetPicker.m

```diff
@@ -34,7 +34,7 @@ CG_INLINE BOOL isIPhone4()
 {
     struct utsname systemInfo;
     uname(&systemInfo);
-
+    
```

Comment:
Here is a trailing spaces. And I can't merge it, unless this file got several trailing spaces.
please, avoid to put spaces in empty lines.